operator_csv_libs/catalog.py

```diff
@@ -164,6 +177,35 @@ def get_bundles_by_substring(self, substring):
         
         #Return the list of bundles matching the substring
         return bundles_matching_substring
+    
+    #Get a single bundle matching a provided string, throw an exception if not found
+    def get_bundle(self, bundle):
+        #Find the bundle, and if not found, throw an exception
+        retrieved_bundle = None
+        for b in self.bundles:
+            if b['name'] == bundle:
+                retrieved_bundle = b
+        if retrieved_bundle == None:
+            raise CatalogError(f"Bundle {bundle} not found in catalog {self.package['name']}")
+        
+        #If it is found, return the channel
+        return retrieved_bundle
+
+    #Function returns a formatted dict containing a channel and its corresponding bundles
+    #The output of this function can be plugged into the add_channel_and_bundles function
+    #in order to add a channel and its corresponding bundles from one catalog to another
+    def get_channel_and_bundles(self, channel):
+        #Get the channel matching the provided channel name
+        channel = self.get_channel(channel)
```

Comment:
Need a try/except block for this. Or the get_channel function could return None instead of throwing an error.